src/sentry/event_manager.py

```diff
@@ -863,6 +867,11 @@ def _get_or_create_release_many(jobs: Sequence[Job], projects: ProjectsMapping)
         if not job["release"]:
             continue
 
+        if options.get("performance.issues.create_issues_through_platform", False) and projects[
+            job["project_id"]
+        ].get_option("sentry:performance_issue_create_issue_through_platform", False):
+            continue
```

Comment:
not sure how this is going to work when there is a mix of n+1 db queries and other types in outputted performance problems. maybe it makes sense to turn group creation through platform on for all issue types at the same time

Reply:
It might be easier. Another option could be to build group creation into the policy layer, but I'm not sure if it's worth it since it's just temporary for this transition.

Reply:
I was adding a check in the wrong place. resolving this since it's no longer applicable